### PR TITLE
Fix #490 - Show loading animation when refreshing arrival times

### LINKED ISSUE #490: Loading animation doesn't appear when refreshing arrival times
**Summary:** 

When you refresh the arrival times, the loading animation doesn't appear.

It does appear when initially loading the arrivals when `ArrivalsListFragment` is first instantiated.

Reported by beta user:

>Animation for the 'Refresh' icon would be nice to have.

**Steps to reproduce:** 

1. Go to any region/stop and view arrivals
1. Tap on the refresh button, or wait for the arrivals to automatically refresh

**Expected behavior:** 

A "loading" message should appear.

**Observed behavior:** 

No "loading" message appears - instead, it looks like nothing happened if the arrivals don't change on refresh.

**Device and Android version:** 

Stock LG G4 w/ Android 6.0, Nexus 5X w/ Android 6.0.1

**Screenshots:** 

Here's what shows on the initial load of arrivals (or at least what I could get in a screenshot):

image

### PR DESCRIPTION
**Pull request by cagryInside**
This Pr implements a new loading dialog when refreshing arrival times.

It is WIP, please `do not` merge.

### CODE REVIEW

**Comment by cagryInside** [comment]
@barbeau Added a new progress dialog view on top of the arrivals list. It looks like this: 
What do you think? I think the color is so bold, we can lighten a little bit.

device-2016-05-11-221923

**Comment by barbeau** [comment]
Yes, maybe keep the same color, but add a little transparency?  I would stick with the theme color, but programmatically adjust the transparency - you can see a demo of this here:
https://github.com/googlemaps/android-maps-utils/blob/master/demo/src/com/google/maps/android/utils/demo/PolySimplifyDemoActivity.java#L59

**Comment by cagryInside** [comment]
The latest commit adds transparency to the `progress bar` and shows another `progress bar`  in the header.

Screenshots:
device-2016-05-12-125229
device-2016-05-12-125259
device-2016-05-12-125446
device-2016-05-12-125945
device-2016-05-12-131927

**Comment by barbeau** [comment]
@cagryInside Screenshots look good!  Did you try this on Gingerbread yet?

File: onebusaway-android/src/main/res/layout/refresh_loading.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2010 Paul Watts (paulcwatts@gmail.com)
```
Comment (barbeau) [inline onebusaway-android/src/main/res/layout/refresh_loading.xml]: Could you please update the copyright header here to 2016 and include USF?

**Comment by barbeau** [comment]
Looks good, thanks @cagryInside!

**Pull request merged**